foreman-cmd

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+FOREMAN_RUBY_PREFIX=${FOREMAN_RUBY_PREFIX:-bundle exec}
+
+run_rails() {
+  [ -n "$RAILS_STARTUP" ] && $RAILS_STARTUP || \
+    [ -n "$BIND" ] && $FOREMAN_RUBY_PREFIX bin/rails server -b $BIND || \
+    $FOREMAN_RUBY_PREFIX bin/rails server
+}
+
+run_webpack() {
+  [ -n "$NODE_ENV" ] && ./node_modules/.bin/webpack-dev-server --config config/webpack.config.js $WEBPACK_OPTS || \
+    env NODE_ENV=development ./node_modules/.bin/webpack-dev-server --config config/webpack.config.js $WEBPACK_OPTS
```

Comment:
How about reusing `foreman` as we already have.
```suggestion
  bundle exec foreman start webpack
```

Reply:
I think you are missing the point. Foreman gem does not emulate full terminal, therefore you cannot use Ruby debuggers or REPL tools. I can't use foreman, I want to avoid this tool.

Yet I understand that for some this is a nice way of starting Foreman development environment and therefore this patch is written in a way that nothing changes for them. `bundle exec foreman start` works just fine.

Example "debugging" session:

```
[lzap@nuc foreman]$ bundle exec foreman start rails
14:49:30 rails.1   | started with pid 37589
14:49:32 rails.1   | 2021-04-28T14:49:32 [I|app|] Rails cache backend: File
14:49:32 rails.1   | => Booting Puma
14:49:32 rails.1   | => Rails 6.0.3.5 application starting in development
14:49:32 rails.1   | => Run `rails server --help` for more startup options
14:49:34 rails.1   |
14:49:34 rails.1   | Frame number: 0/75
14:49:34 rails.1   |
14:49:34 rails.1   | From: /home/lzap/work/foreman/config/initializers/apipie.rb:5 :
14:49:34 rails.1   |
14:49:34 rails.1   |      1: require 'find'
14:49:34 rails.1   |      2: require "pry"
14:49:34 rails.1   |      3: binding.pry
14:49:34 rails.1   |      4:
14:49:34 rails.1   |  =>  5: ApipieDSL.configure do |config|
14:49:34 rails.1   |      6:   config.default_version = 'v1'
14:49:34 rails.1   |      7:   config.app_name = 'Foreman'
14:49:34 rails.1   |      8:   config.app_info = 'The Foreman is aimed to be a single address for all machines life cycle management.'
14:49:34 rails.1   |      9:   config.doc_base_url = '/templates_doc'
14:49:34 rails.1   |     10:   config.markup = ApipieDSL::Markup::Markdown.new if Rails.env.development? && defined? Maruku
14:49:34 rails.1   |
[1] pry(main)> 1   |
[2] pry(main)> 1   |
[3] pry(main)> 1   |
[4] pry(main)> rte |
NameError: undefined local variable or method `rte' for main:Object
14:49:39 rails.1   | Did you mean?  true
14:49:39 rails.1   | from (pry):1:in `<main>'
```

If the output does not make sense to you, that is correct. I was trying enter a command and then exit the session.

Reply:
I agree with above and I'd like to go the other direction - deprecating and removing `foreman` gem as a standard, its only value IMHO is we have fun starting foreman through foreman command, but it confuses rails developers who don't know `foreman`, nor `theforeman` and it prevents them from effectively debugging, that can really ruin the developer experience!

Reply:
I do agree with you that foreman gem does not necessarily bring much value other than it is a convenient way of starting. I do not want to do it in this PR tho, however, if you actually think about this then I can provide "foreman-cmd run" command that would do exactly do the same - run both commands in the foreground.